org.gnome.PowerStats.json

```diff
@@ -0,0 +1,68 @@
+{
+  "app-id": "org.gnome.PowerStats",
+  "sdk": "org.gnome.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "command": "gnome-power-statistics",
+  "finish-args": [
+    "--share=ipc",
+    "--system-talk-name=org.freedesktop.UPower"
+  ],
+  "cleanup": [
+     "*.la",
+    "/include",
+    "/share/man"
+  ],
+  "modules": [
+    "shared-modules/udev/udev-175.json",
+    {
+      "name": "libusb",
+      "config-opts": [
+        "--disable-static"
+      ],
+      "cleanup": [
+        "/lib/pkgconfig"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/libusb/libusb/releases/download/v1.0.22/libusb-1.0.22.tar.bz2",
+          "sha256": "75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157"
+        }
+      ]
+    },
+    {
+      "name": "upower",
+      "cleanup": [
+        "/bin/upower",
+        "/etc/dbus-1",
+        "/etc/UPower",
+        "/libexec",
+        "/libexec/rules.d",
+        "/lib/girepository-1.0",
+        "/lib/pkgconfig",
+        "/share/dbus-1",
+        "/share/gir-1.0",
+        "/var"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://upower.freedesktop.org/releases/upower-0.99.7.tar.xz",
+          "sha256": "24bcc2f6ab25a2533bac70b587bcb019e591293076920f5b5e04bdedc140a401"
+        }
+      ]
+    },
+    {
+      "name": "gnome-power-manager",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ftp.acc.umu.se/pub/GNOME/sources/gnome-power-manager/3.30/gnome-power-manager-3.30.0.tar.xz",
```

Comment:
Why `https://ftp.acc.umu.se/` instead of `https://ftp.gnome.org/`?

Full URL: https://ftp.gnome.org/pub/GNOME/sources/gnome-power-manager/3.30/gnome-power-manager-3.30.0.tar.xz

Reply:
Fixed, thanks.